config.go

```diff
@@ -73,15 +73,25 @@ func FieldTag(fieldName, tagName, helpTxt string) *GoConfig {
 
 // New creates a new config.
 func New() *GoConfig {
+	return NewWithPrefix("")
+}
+
+// NewWithPrefix creates a new config with a global prefix for loaders.
+func NewWithPrefix(prefix string) *GoConfig {
+	if prefix != "" {
+		prefix = util.ToLowerSnake(prefix)
```

Comment:
Why would this always be lower snake case? Doesn't it get formatted later in the respective flag and env loader?